addons/web/controllers/main.py

```diff
@@ -1668,7 +1668,7 @@ def get_fonts(self, fontname=None):
         else:
             current_dir = os.path.dirname(os.path.abspath(__file__))
             fonts_directory = os.path.join(current_dir, '..', 'static', 'src', 'fonts', 'sign')
-            font_filenames = sorted(os.listdir(fonts_directory))
+            font_filenames = sorted(glob.glob('%s/*.[ot]tf' % fonts_directory))
```

Comment:
Do we still need to impose TTF in 2020? There are formats like WOFF2 that [are supported](https://developer.mozilla.org/en-US/docs/Web/Guide/WOFF#Browser_compatibility) by all browsers now and are much lighter.

Reply:
Ok, I'll check if it works with woff2.

Reply:
It works with woff and woff2